pylint/lint/utils.py

```diff
@@ -46,11 +47,27 @@ def prepare_crash_report(ex: Exception, filepath: str, crash_file_path: str) ->
 {file_content}
 ```
 
-pylint crashed with a ``{ex.__class__.__name__}`` and with the following stacktrace:
+### Command used
+
+```shell
+pylint a.py
 ```
+
+### Pylint output
+
+<details open>
+    <summary>
+        pylint crashed with a ``{ex.__class__.__name__}`` and with the following stacktrace:
+    </summary>
+
+```python
```

Comment:
A code block can't be in ``details``. I'd prefer the codeblock over the minimizing.

Reply:
Are you sure ? It works in #8509 (with the proper amount of line break)

Reply:
TIL! This was broken for years I think

Reply:
Ho it was broken 5mn ago too, I edited the thing 5 times 😄 Adding line breaks helps it seems.